TestCases/rans/oneram6/turb_ONERAM6_nk.cfg

```diff
@@ -18,22 +18,29 @@ NEWTON_KRYLOV= YES
 
 % Iterations and tolerance for the Krylov part, it is important not to
 % "over solve", tolerance should be as high as possible.
-LINEAR_SOLVER_ITER= 5
-LINEAR_SOLVER_ERROR= 0.25
+LINEAR_SOLVER_ITER= 8
+LINEAR_SOLVER_ERROR= 0.2
 
 % For "n0" iterations or "r0" residual reduction, the normal quasi-Newton iterations
 % are used. Then, they become the preconditioner for the NK iterations with "np" linear
 % iterations or "tp" tolerance, with "np"=0 the linear preconditioner (e.g. ILU) is
 % used directly (this may be enough for unsteady).
 % The tolerance for NK iterations is initially relaxed by factor "ft", and reaches
 % LINEAR_SOLVER_ERROR after "rf" residual reduction (additional to "r0").
+% If "ft"=1, "rf" does not matter.
 % The Jacobian-free products are based on finite differences with step "e".
-NEWTON_KRYLOV_IPARAM= (10, 3, 2) % n0, np, ft
-NEWTON_KRYLOV_DPARAM= (1.0, 0.1, -6.0, 1e-5) % r0, tp, rf, e
+% "rnk" makes the linear system easier to solve by relaxing MUSCL (just for the NK
+% matrix-free system). 0 is the same as not using NK, 1 is full MUSCL, a negative
+% values means "automatic", the solver changes the value so that the linear system
+% can be solved.
+% If "n0">0, and RESTART_SOL=NO, the NK relaxation is applied to the residual (not
+% just to the NK system) to ramp from MUSCL=OFF to ON during "n0" iterations.
+NEWTON_KRYLOV_IPARAM= (200, 0, 1) % n0, np, ft
+NEWTON_KRYLOV_DPARAM= (0, 0, 0, 1e-5, -1) % r0, tp, rf, e, rnk
 
 CFL_ADAPT= YES % it's needed
 CFL_NUMBER= 10
-CFL_ADAPT_PARAM= ( 0.8, 1.1, 5, 1000 ) % no point using NK with low CFL values
+CFL_ADAPT_PARAM= ( 1, 1.05, 10, 400 ) % no point using NK with low CFL values
```

Comment:
so when the linear solver convergence stalls, we don't reduce cfl but keep it at the current level, and that is sufficient? Often/in general?